ui/src/project.ts

```diff
@@ -321,6 +493,56 @@ export const repositoryPathValidationStore = (
   }
 };
 
+const VALID_PEER_MATCH = /^[\w\d]{54}$/;
```

Comment:
Let's make sure this regex ony matches valid `base32`.

Reply:
Nooooo base58 hahaha https://github.com/radicle-dev/radicle-link/blob/master/spec/radicle-link.md#cryptographic-identities

Reply:
What @FintanH said.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1014/commits/b5098322f2d4453b6229c85827c57814645aae52